### ISSUE TITLE
StravaBestRunningEffortId needs to be a Long

### ISSUE THREAD

**Issue by kcfwpi** [danshannon/javastravav3api#183]
The ID variable in the StravaBestRunningEffortId class needs to be a Long. It's currently an Integer.

**Comment by hobovsky** [comment]
No, it's not. Okay, it is. Kinda. https://github.com/danshannon/javastravav3api/issues/180
